PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudStreamingLogConsoleView.java

```
@@ -64,21 +64,16 @@ public void startLog(InputStream inputStream) {
         this.print("Streaming Log Start.\n", ConsoleViewContentType.SYSTEM_OUTPUT);
         executorService = Executors.newSingleThreadExecutor();
         executorService.submit(() -> {
-            final BufferedReader br = new BufferedReader(new InputStreamReader(logInputStream, StandardCharsets.UTF_8));
-            while (true) {
-                try {
+            try (final BufferedReader br = new BufferedReader(new InputStreamReader(logInputStream,
+                                                                                    StandardCharsets.UTF_8));) {
+                while (enable.get()) {
                     final String log = br.readLine();
-                    if (enable.get()) {
-                        SpringCloudStreamingLogConsoleView.this.print(log + "\n", ConsoleViewContentType.NORMAL_OUTPUT);
-                    } else {
-                        break;
-                    }
+                    SpringCloudStreamingLogConsoleView.this.print(log + "\n", ConsoleViewContentType.NORMAL_OUTPUT);
                     Thread.sleep(50);
-                } catch (IOException e) {
-                    // swallow io exception
-                } catch (InterruptedException e) {
-                    break;
                 }
+            } catch (IOException | InterruptedException e) {
+                // swallow Exception
+                e.printStackTrace();
```

If swallow Exception, should not print stack trace